### PR TITLE
fix(DB/broadcast_text_locale): Guard gossip for hunter class [esES/esMX]

### PR DESCRIPTION
**Pull request by pangolp**
## CHANGES PROPOSED:
-  Message to Spanish corrected

## ISSUES ADDRESSED:
- Closes 

## TESTS PERFORMED:
WoWScrnShot_101420_095343

WoWScrnShot_101420_101509

WoWScrnShot_101420_101528

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Kitzunu** [comment]
Why close

**Comment by pangolp** [comment]
I particularly closed it because I got really tired of waiting 5 days, seeing how they approve other things and merge them much faster. I am aware that this "fix" does not change your life, but as I see that it is not part of your priorities, I decided to close it, but you are free to reopen it if you wish.

**Comment by pangolp** [comment]
I even took the trouble to upload the photos, where I show them the before and after. I have seen pull requests, where they are not able to give information of anything, and they have approved them, but well, evidently who is in charge of approving or not the contributions, has other priorities and I think it is good. I wish them a lot of luck.

**Comment by FrancescoBorzi** [comment]
@pangolp we are all in charge of finding someone to test stuff, including you :)

**Comment by pangolp** [comment]
The pull request is yours, the code is free. I didn't think that a simple **UPDATE** would take 1 week to be tested. That's why I closed it, but don't worry, you won't see any more pull request from me. I admit that the mistake was mine all the same. Thanks for all this time sharing, I really appreciate it.

File: data/sql/updates/pending_db_world/rev_1602681411978701400.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1602681411978701400');
+
+UPDATE `broadcast_text_locale` SET `MaleText`='Cazador', `FemaleText`='Cazadora' WHERE `ID`=50546 AND `locale` IN ('esES', 'esMX');
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1602681411978701400.sql:3]:
```suggestion
UPDATE `broadcast_text_locale` SET `MaleText`='Cazador', `FemaleText`='Cazadora' WHERE `ID`=50546 AND `locale` IN ('esES', 'esMX');


```

**Review by Si1ker — APPROVED** [review]

**Pull request merged**